core-skills/frontend-web-development/resources/semantic-html.md

```diff
@@ -1,8 +1,17 @@
-https://en.wikipedia.org/wiki/Semantic_HTML
-https://developer.mozilla.org/en-US/docs/Web/HTML
-https://codelabs.developers.google.com/codelabs/autocomplete/index.html
-http://html5pattern.com/
-https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/How_to_structure_an_HTML_form
-http://alistapart.com/article/sensibleforms
-http://html5doctor.com/html5-forms-input-types/
-http://codeguide.co/#html
+# Resources on Semantic HTML
+
+Semantic HTML is the use of HTML markup emphasising the _meaning_ of the information that it contains.
+
+This is in contrast to markup that focuses on _presentation_ or _visual structure_.
+
+For example, a `<li>` (list) element shouldn't be used because you want bullet points - it should be used because you have items that _make sense_ in a list.
+
+For a second example, notice the introduction of `<strong>` and `<em>` tags, preferred over `<b>` and `<i>`, because they describe the intent - "I want to emphasis this content" rather than "I want this content to look a certain way".
```

Comment:
Should mention the HTML5 block elements, and the semantic meaning they add, and why they are preferred to `div` soup or link to an article

Reply:
Good idea, I'll look for a good resource.